packages/common/src/testUtil/testSubsetGrep.txt

```diff
@@ -0,0 +1,12 @@
+# This file contains the grep strings to pass to Mocha when running a subset of tests.
+# These grep strings will be used with the "Run Single Extension Test" launch configuration.
```

Comment:
"Run test subset"

Reply:
Thanks. silly of me to copy paste stale doks without updating them. I will change this once there is clarity about what overall approach we want to take.